device-types/Extreme Networks/7520-48XT-6C.yaml

```diff
@@ -0,0 +1,132 @@
+---
+manufacturer: Extreme Networks
+model: 7520-48XT-6C
+slug: extreme-networks-7520-48xt-6c
+part_number: 7520-48XT-6C
+u_height: 1
+weight: 7.41
+weight_unit: kg
+is_full_depth: true
+comments: '[Extreme Networks 7520 Data Sheet](https://extr-p-001.sitecorecontenthub.cloud/api/public/content/b5da59835f5d4d10b740208284c8bc09?v=9df9bc2b)'
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: Storage
```

Comment:
Please don't model generic USB ports that aren't used for console connections.

Reply:
Thanks, I have removed those Storage USB ports.